src/test/java/unit/WorkspaceOverride.java

```diff
@@ -390,4 +395,39 @@ void workspace() throws IOException {
     matchingWorkspaces = listWorkspacesWithId(workspace1.id);
     assertEquals(1, matchingWorkspaces.size(), "workspace 1 is still included in list");
   }
+
+  @Disabled // TODO (PF-869): enable this test once polling notebook operations is fixed
+  @DisplayName("notebook commands respect workspace override")
```

Comment:
optional: seems kind of like overkill to have to include every workspace id command in this test file. Consider testing that the workspace id overrides alongside the other tests for the same command. Its's still a test of the notebooks commands, rather than a test of the mixin.

Reply:
You're right that it's testing the notebooks commands, instead of the mixin itself. I grouped these tests together because they have shared setup (i.e. create two workspaces to switch between and clean them up afterwards).